### ISSUE TITLE
Idk why but the raven coin isnt mining. Is there issue with my setup

### ISSUE THREAD

**Issue by JarmoDrn** [UnamSanctam/SilentCryptoMiner#2050]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/154092299/f94c40d3-c897-4c8e-afdd-fca0e43a1cac)

**Comment by UnamSanctam** [comment]
That looks correct (I assume it's showing the last part of your wallet address), can you run the checker that got built together with your miner and post a screenshot of that?

**Comment by JarmoDrn** [comment]
> That looks correct (I assume it's showing the last part of your wallet address), can you run the checker that got built together with your miner and post a screenshot of that?

yeah its showing last part of my wallet address. It wont show anything on my checker since im running in my vm. but I sent it to one guy and he ran it (I saw him run it) but it doesnt show any sign in 2miners dashboard

**Comment by JarmoDrn** [comment]
also gotta ask. since ravencoin is mining on their GPU can I add monero miner to the same executable to be mined on their CPU or willl that crash it or something

**Comment by UnamSanctam** [comment]
>yeah its showing last part of my wallet address. It wont show anything on my checker since im running in my vm. but I sent it to one guy and he ran it (I saw him run it) but it doesnt show any sign in 2miners dashboard

And his GPU has enough free VRAM to mine? 2miners also has a high share difficulty so finding a share can take a while depending on the hashrate.

>also gotta ask. since ravencoin is mining on their GPU can I add monero miner to the same executable to be mined on their CPU or willl that crash it or something

Yes that will work fine, and is also what the miner is built for. CPU mining and GPU mining use different kinds of resources.

**Comment by JarmoDrn** [comment]
> ?

so maybe i should change pools? the guy who ran it had  Intel(R) HD Graphics 4600

**Comment by UnamSanctam** [comment]
Oh, well that's not a real GPU (a dedicated GPU), that's just the integrated graphics in his CPU.

**Comment by JarmoDrn** [comment]
> Oh, well that's not a real GPU (a dedicated GPU), that's just the integrated graphics in his CPU.

oh i see. so it needs to be a dedicated gpu in order for it to be able to mine GPU coins

**Comment by UnamSanctam** [comment]
Yes a dedicated GPU with at least 4.2GB of free VRAM is needed to mine Ravencoin. While it's theoretically possible to mine on some CPU integrated graphics by using the RAM the hashrate would be very low, and could cause lag.
